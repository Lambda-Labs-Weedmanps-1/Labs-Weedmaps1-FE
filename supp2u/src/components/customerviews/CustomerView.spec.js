import React from 'react';
import { mount, configure, shallow, enzyme } from 'enzyme';
import { MemoryRouter } from 'react-router';
import Adapter from 'enzyme-adapter-react-16';

// import Customer from './AddCustomerPresentation'
import View from './CustomerView';

configure({ adapter: new Adapter() });

describe('Customer view Renders', () => {
    // If Map component render is ever non functional, this should fail
    it('should render correctly when component called', () => {
      const cust = shallow(<View />);

      expect(cust).toMatchSnapshot();
      expect(cust.exists()).toBe(true)
    });
});