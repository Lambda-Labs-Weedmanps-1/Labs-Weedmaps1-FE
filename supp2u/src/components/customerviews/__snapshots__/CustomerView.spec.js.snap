// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Customer view Renders loading message displays when there is no picture 1`] = `"loading"`;

exports[`Customer view Renders should render correctly when component called 1`] = `
<div
  className="c-view"
>
  <p>
    Image loading....
  </p>
  <h1
    className="name-box"
  />
  <div>
    <h3>
      Your Reviews
    </h3>
    <div
      className="review-box"
    >
      <p />
      <p />
      <p />
    </div>
  </div>
</div>
`;
